libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerMixpanel.java

```diff
@@ -384,6 +384,12 @@ private AnalyticsTrackerMixpanelInstructionsForStat instructionsForStat(
                 instructions.setSuperPropertyAndPeoplePropertyToIncrement("number_of_times_unfollowed_reader_tag");
                 instructions.setCurrentDateForPeopleProperty("last_time_unfollowed_reader_tag");
                 break;
+            case READER_SEARCH_LOADED:
+                instructions = AnalyticsTrackerMixpanelInstructionsForStat.
+                        mixpanelInstructionsForEventName("Reader - loaded search");
```

Comment:
Probably not a big deal, but I noticed other Mixpanel event names are word capitalized.